docs/data/tree-view/simple-tree-view/customization/customization.md

```diff
@@ -11,11 +11,51 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
 
 <p class="description">Customize the rendering of your Tree View.</p>
 
-## Custom icons, border and animation
+## Basics of customization
+
+### Custom icons
+
+Use the `defaultCollapseIcon`, `defaultExpandIcon`, and `defaultEndIcon` props to customize the Tree View icons. You can use either icon libraries, such as Material Icons, or create your own.
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Custom toggle animations
+
+Use the `TransitionComponent` prop on the `TreeItem` to pass a component that handles your animation.
+
+{{"demo": "CustomAnimation.js", "defaultCodeOpen": false}}
+
+### Styling
+
+You can use the `treeItemClasses` to target the internal elements of the TreeView.
```

Comment:
```suggestion
Use the `treeItemClasses` to target internal elements of the Tree Item component and change their styles.
```

PS: the demo looks a bit muddy in dark mode — maybe we should add the specific styles for it? It seems it's just light mode-focused.

<img width="250" alt="Screenshot 2024-01-12 at 08 02 20" src="https://github.com/mui/mui-x/assets/67129314/373a896d-628c-4118-b08c-e6a16f5cac7e">